perfkitbenchmarker/providers/aws/aws_emr.py

```diff
@@ -61,19 +62,35 @@ def __init__(self, spark_service_spec):
       self.machine_type = DEFAULT_MACHINE_TYPE
     self.cmd_prefix = util.AWS_PREFIX + ['emr']
     if FLAGS.zones:
-      region = util.GetRegionFromZone(FLAGS.zones[0])
-      self.cmd_prefix += ['--region', region]
+      self.region = util.GetRegionFromZone(FLAGS.zones[0])
+    else:
+      cmd = ['aws', 'configure', 'get', 'region']
```

Comment:
If the default region is already set then you shouldn't need to explicitly set it. And if it isn't set, then "aws configure get region" doesn't return anything.